commit a93a2ab1918630c6d571b5a24379c15a0458d1fa
Author: Martin Gräßlin <mgraesslin@kde.org>
Date:   Wed Jan 28 16:20:57 2015 +0100

    Disable libinput integration if >= 0.8 is found
    
    libinput 0.8 is incompatible causing the build to fail. As we are in
    dependency freeze the only option is to disable the build.
    
    BUG: 342893
    FIXED-IN: 5.2.1

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 550ef57..8d9c593 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -145,7 +145,7 @@ set_package_properties(UDev PROPERTIES  URL "http://www.freedesktop.org/software
                                         PURPOSE "Required for input handling on Wayland."
                         )
 set(HAVE_INPUT FALSE)
-if (Libinput_FOUND AND UDEV_FOUND)
+if (Libinput_FOUND AND UDEV_FOUND AND Libinput_VERSION VERSION_LESS 0.8)
     set(HAVE_INPUT TRUE)
 endif()
 
